api/v1alpha1/webhook_suite_test.go

```diff
@@ -24,11 +24,17 @@ import (
 	"testing"
 	"time"
 
+	"github.com/aws/eks-anywhere-packages/api/v1alpha1"
```

Comment:
Looks like some things need linting?

Reply:
Yea it seems like I need to configure something for imports in my environment.

Reply:
Yeah, I've never used as many separate tools to do this, as we use here, but this is what I'm using presently:

```sh
GOPATH="${HOME}/go"
local="$(go list -m)"
gofmt -s -w "$@"
goimports -w -local "$local" "$@"
gci -w -local "$local" "$@"
```

Reply:
Yea I actually had to downgrade gci too I believe.